src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.legal.strings

```diff
@@ -121,14 +121,14 @@
 
 "errorReport_Legal_DataPrivacy_Bullet5" = "Private individuals as well as companies are not allowed to make you share a recorded error report with them.";
 
-"errorReport_Legal_Consent_Headline" = "Your consent";
+"errorReport_Legal_SendReports_Headline" = "Your consent";
 
-"errorReport_Legal_Consent_Intro" = "By tapping on “Accept and Send”, you consent to the following steps:";
+"errorReport_Legal_SendReports_Subline" = "By tapping on “Accept and Send”, you consent to the following steps:";
 
-"errorReport_Legal_Consent_Bullet1_Header" = "The error report you create will be sent to the RKI for error analysis purposes.";
+"errorReport_Legal_SendReports_Bullet1_Part1" = "The error report you create will be sent to the RKI for error analysis purposes.";
 
-"errorReport_Legal_Consent_Bullet1_Paragraph" = "The RKI will analyse the error report in order to be able to identify and fix possible causes of the errors that occur in your app. If you provide the error report ID to the technical support team, then the RKI will be able to associate the error report with you personally.";
+"errorReport_Legal_SendReports_Bullet1_Part2" = " The RKI will analyse the error report in order to be able to identify and fix possible causes of the errors that occur in your app. If you provide the error report ID to the technical support team, then the RKI will be able to associate the error report with you personally.";
```

Comment:
Really want to add that space?
```suggestion
"errorReport_Legal_SendReports_Bullet1_Part2" = "The RKI will analyse the error report in order to be able to identify and fix possible causes of the errors that occur in your app. If you provide the error report ID to the technical support team, then the RKI will be able to associate the error report with you personally.";

```

Reply:
Good point. Changed the implementation in class SendErrorLogsViewModel to add the necessary space.